validate.groovy

```diff
@@ -19,9 +19,105 @@ class ValidateSpecification extends Specification {
         schema = SchemaLoader.load(rawSchema)
     }
 
-    def 'this should throw an exception'() {
+    def 'qbicID exception'() {
         when:
-        schema.validate(new JSONObject(new File("./invalid_example.json").text)) // throws a ValidationException if this object is invalid
+        schema.validate(new JSONObject(new File("./invalid_qbicId.json").text)) // throws a ValidationException if this object is invalid
+
+        then:
+        org.everit.json.schema.ValidationException exception = thrown(org.everit.json.schema.ValidationException)
+    }
+
+    def 'seqFacilityId exception'() {
```

Comment:
More expressive description of the test method please, for all of them :)

Reply:
solved by b297383